drop/modules/aberrant-expression-pipeline/OUTRIDER/Summary.R

```diff
@@ -125,19 +125,13 @@ if (nrow(res) > 0) {
 
 
 #' ## Results table
-#+echo=F
+
+## Save results table in html folder and provide link to download
+file <- gsub(".html$", ".tsv", snakemake@output$wBhtml)
+fwrite(res, file, sep = '\t', quote = F)
+#+ echo=FALSE, results='asis'
+cat(paste0("<a href='./", basename(file), "'>Download OUTRIDER results table</a>"))
```

Comment:
Is it guaranteed that the download file is in the same directory as the html output? Maybe add a comment

Reply:
it's guaranteed